File: src/com/starflask/terminal/SubMenus/TerminalMenuOption.java
package com.starflask.terminal.SubMenus;

public class TerminalMenuOption {
	
	String title;
	
	public TerminalMenuOption(String title) {
		 this.title=title;
	}

	public String render() {
		 
		return title;
	}
	
	
	public void execute(){}

}

File: src/com/starflask/terminal/SubMenus/TerminalMenuOption.scala
package com.starflask.terminal.SubMenus

class TerminalMenuOption {
  var title = new String("no title");
	
	def TerminalMenuOption( title:String) {
		 this.title=title;
	}

	def  render():String = {
		 
		 title 
	}
	
	
	def  execute(){}

}